frontend-student/src/components/home/Home2.jsx

```
@@ -0,0 +1,54 @@
+        </div>
+      </div>
+
+      {/* Overlay with Hero Section */}
+      <div className="absolute inset-0 bg-gradient-to-r from-black/70 to-black/30 text-white flex items-center justify-center z-10">
+        <div className="text-center px-6">
+          <h1 className="text-4xl md:text-6xl font-bold mb-4">EduNexus</h1>
+          <h4 className="text-xl md:text-2xl font-medium mb-2">
+            Connecting Minds, Empowering Futures.
+          </h4>
+          <p className="text-md md:text-lg mb-6">A Project by Team React Fusion</p>
+          <button className="btn btn-primary btn-lg normal-case text-white">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)